src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace.cs

```diff
@@ -193,6 +194,7 @@ private static IList<Exception> Flatten(ICollection<Exception> exceptions)
             var aggregate = new AggregateException(exceptions);
             return aggregate.Flatten().InnerExceptions
                 .Select(UnwrapException)
+                .Where(ex => !(ex is JoinableTaskContextException))
```

Comment:
If we don't care about this case, or its going away in @rchande 's PR then I suggest we merge this once the azure VMs are updated so we can begin inserting again.  We can remove this once @rchande 's changes make it to master